compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirErrors.kt

```diff
@@ -164,9 +172,13 @@ object FirErrors {
     val USELESS_VARARG_ON_PARAMETER by warning0<FirSourceElement, KtParameter>()
 
     // Properties & accessors
-    val ABSTRACT_PROPERTY_IN_NON_ABSTRACT_CLASS by error2<FirSourceElement, KtModifierListOwner, FirMemberDeclaration, FirMemberDeclaration>(SourceElementPositioningStrategies.MODALITY_MODIFIER)
+    val ABSTRACT_PROPERTY_IN_NON_ABSTRACT_CLASS by error2<FirSourceElement, KtModifierListOwner, FirMemberDeclaration, FirMemberDeclaration>(
+        SourceElementPositioningStrategies.MODALITY_MODIFIER
+    )
     val PRIVATE_PROPERTY_IN_INTERFACE by error0<FirSourceElement, KtProperty>(SourceElementPositioningStrategies.VISIBILITY_MODIFIER)
 
+    val WRONG_SETTER_PARAMETER_TYPE by error0<FirSourceElement, KtProperty>(SourceElementPositioningStrategies.DEFAULT)
```

Comment:
In FE 1.0 this diagnostic has two parameters with expected and actual setter parameter type. Please add them here too

See `Errors.WRONG_SETTER_PARAMETER_TYPE`